src/pages/dashboard/dashboard.component.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+
+const Dashboard = () => {
+  const currentUser = useSelector((state) => state.currentUser);
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const handleSignOut = () => {
+    dispatch({
+      type: '@currentUser/SIGN_OUT',
```

Comment:
What do you think about create a file exporting the constants instead working with strings directly?